packages/framework-provider-azure-infrastructure/src/infrastructure/index.ts

```
@@ -8,11 +8,9 @@ import {
   createWebSiteManagementClient,
 } from './setup'
 
-export const deploy = (configuration: BoosterConfig, logger: Logger): Promise<void> =>
-  deployApp(logger, configuration).catch(logger.error)
```

this catch mas masking all the errors, we were debugging kind of blindly.